libs/langgraph/langgraph/pregel/read.py

```diff
@@ -165,9 +170,21 @@ def __init__(
         self.retry_policy = retry_policy
         self.tags = tags
         self.metadata = metadata
+        if self.bound is not DEFAULT_BOUND:
+            try:
+                subgraph = find_subgraph_pregel(self.bound)
+            except Exception:
+                subgraph = None
+            if subgraph:
+                self.subgraphs = [subgraph]
+            else:
+                self.subgraphs = []
+        else:
+            self.subgraphs = []
 
     def copy(self, update: dict[str, Any]) -> PregelNode:
         attrs = {**self.__dict__, **update}
+        attrs.pop("subgraphs")
```

Comment:
I'm guessing no need for a `None` here

Reply:
I think we actually always assign to subgraphs?